testsuite/test_wbemconnection_mock.py

```diff
@@ -1031,6 +1032,46 @@ def test_compile_qualifiers(self, conn, ns, tst_qualifiers_mof):
         quals = conn.EnumerateQualifiers(namespace=ns)
         assert len(quals) == 8
 
+    def test_unicode(self, conn):
+        """
+        Test compile and display repository with unicode in mof string
+        """
+        ns = 'root/blah'
+
+        qmof = """
+        Qualifier Description : string = null,
+            Scope(any),
+            Flavor(EnableOverride, ToSubclass, Translatable);
+
+        Qualifier Key : boolean = false,
+            Scope(property, reference),
+            Flavor(DisableOverride, ToSubclass);
+        Qualifier Description : string = null,
+            Scope(any),
+            Flavor(EnableOverride, ToSubclass, Translatable);
+        """
+
+        cmof = u"""
+        class CIM_Foo {
+                [Key,
+                 Description ("Å \u0420\u043e\u0441\u0441\u0438\u044f"
```

Comment:
Please avoid having unicode characters directly in the source. Use `\u212b` instead.